Xsup 39289 auto resolve (#36028)
demisto/content: Packs/Core/ReleaseNotes/3_0_59.md

#### Integrations

##### Indicators detection

> - Updated the CoreIRApiModule with support for for status ***resolved_auto_resolve***  close-reason in XSOAR-XDR close-reasons mapping.

Reviewer: ```suggestion
- Updated the CoreIRApiModule to support status ***resolved_auto_resolve***  close-reason in XSOAR-XDR close-reasons mapping.
```

##### Investigation & Response

> - Updated the CoreIRApiModule with support for for status ***resolved_auto_resolve***  close-reason in XSOAR-XDR close-reasons mapping.

Reviewer: Same as above
